services/api/src/lib/evaluate/agents_with_vision.py

```
@@ -522,14 +932,69 @@ def _create_unified_property(
+                    original_price = total
+
+                    # Handle both running and non-running event loop cases
+                    if not loop.is_running():
+                        # If no loop is running, we can use run_until_complete
+                        total = loop.run_until_complete(
+                            self._convert_booking_price_with_llm(property_data, total)
+                        )
+                    else:
+                        # If a loop is already running, we need a different approach
+                        import concurrent.futures
+                        import functools
```

Ruff (0.8.2)

`functools` imported but unused

Remove unused import: `functools`

(F401)